events/README_SQS.md

```diff
@@ -1,7 +1,7 @@
 
 # Sample Function
 
-The following is a sample class and Lambda function that receives Amazon SQS event message data as input, writes some of the message data to CloudWatch Logs, and responds with a 200 status and the same body as the request. (Note that by default anything written to Console will be logged as CloudWatch Logs events.)
```

Comment:
Appreciate the docs changes. But they aren't related to the ClientVPN feature. Can you split them into a separate PR?

Reply:
I have cherry-picked the other changes to a separate branch and will shortly push those changes